java/arcs/core/storage/DirectStoreMuxer.kt

```diff
@@ -120,6 +126,9 @@ class DirectStoreMuxer<Data : CrdtData, Op : CrdtOperation, T>(
     suspend fun store(id: String) = storeMutex.withLock {
         stores.getOrPut(id) {
             setupStore(id)
+        }.also {
+            val toClose = closeList.getAndSet(emptyList())
+            toClose.forEach { closeStore(it) }
```

Comment:
Feel there is a performance optimization task here: asynchronous cache eviction.
In the long run, creating a new Store or just getting an existing Store will incur eviction overhead.
This happens on existing LruCahceMap as well, just note an optimization opportunity here.

Reply:
The KDoc of LruCacheMap speaks of adding a periodic cleanup task later.

Reply:
This definitely crossed my mind! I think it needs to happen in tandem with better coroutine scope/context mgmt up and down the storage stack.